plot-lib/plot/private/common/types.rkt

```diff
@@ -162,3 +162,18 @@
    [draw-pict (->* [pict (Vectorof Real)] (Anchor Real) Void)]
    [calculate-legend-rect (-> (Listof legend-entry) Rect Anchor Rect)]
    [draw-legend (-> (Listof legend-entry) Rect Void)]))
+
+(define-type Plot-Metrics<%>
+  (Class
+   [get-plot-bounds (-> (Vectorof (Vector Real Real)))]
+   [plot->dc (-> (Vectorof Real) (Vectorof Real))]))
+
+(struct plotpict pict ([bounds : (Vectorof (Vector Real Real))]
```

Comment:
I think the structure should be named `plot-pict` as this is more consistent with other names.

Reply:
oops, `plot-pict` is a function name...